### PR TITLE
chore: Implement prettier

### PR DESCRIPTION
**Pull request by Koenkk**
Implements [prettier](https://prettier.io/) for better and automatic code formatting

### CODE REVIEW

File: lib/extension/networkMap.ts [hunk 1]
```diff
@@ -113,35 +120,31 @@ export default class NetworkMap extends Extension {
...
+                style = `style="rounded, filled", fillcolor="${colors.fill.router}", ` + `fontcolor="${colors.font.router}"`;
             } else {
-                style = `style="rounded, dashed, filled", fillcolor="${colors.fill.enddevice}", ` +
-                    `fontcolor="${colors.font.enddevice}"`;
+                style = `style="rounded, dashed, filled", fillcolor="${colors.fill.enddevice}", ` + `fontcolor="${colors.font.enddevice}"`;
             }
 
             // Add the device with its labels to the graph as a node.
-            text += `  "${node.ieeeAddr}" [`+style+`, label="{${labels.join('|')}}"];\n`;
+            text += `  "${node.ieeeAddr}" [` + style + `, label="{${labels.join('|')}}"];\n`;
```
Comment (Nerivec) [inline lib/extension/networkMap.ts:131]:
```typescript
text += `  "${node.ieeeAddr}" [${style}, label="{${labels.join('|')}}"];\n`;
```

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "semi": true,
+    "trailingComma": "all",
+    "singleQuote": true,
+    "printWidth": 150,
+    "bracketSpacing": false,
+    "tabWidth": 4
+}
```
Comment (Nerivec) [inline .prettierrc:8]: I suggest adding this as well:
```
endOfLine: "lf"
```

Might want to add https://github.com/prettier/eslint-config-prettier for better eslint support.

File: lib/controller.ts [hunk 3]
```diff
@@ -111,7 +140,9 @@ export class Controller {
             this.eventBus.onAdapterDisconnected(this, this.onZigbeeAdapterDisconnected);
         } catch (error) {
             logger.error('Failed to start zigbee');
-            logger.error('Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions'); /* eslint-disable-line max-len */
+            logger.error(
+                'Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions',
+            ); /* eslint-disable-line max-len */
```
Comment (Nerivec) [inline lib/controller.ts:145]: Probably should do some whole-project searching to remove these no-longer-needed eslint disablers.

File: package.json [hunk 4]
```diff
@@ -23,6 +23,8 @@
     "build": "tsc && node index.js writehash",
     "build-watch": "tsc --watch",
     "eslint": "eslint lib/ --max-warnings=0",
+    "pretty:write": "prettier --write lib",
```
Comment (Nerivec) [inline package.json:26]: write/check in `test` too (eventually they will get refactored to TS too 😜)?

File: lib/controller.ts (hunk 3 above)
Comment (Koenkk) [reply lib/controller.ts:145]: Removed all of them

File: package.json (hunk 4 above)
Comment (Koenkk) [reply package.json]: Good idea, it also works fine for JS

**Pull request merged**